replay/model_handler.py

```diff
@@ -64,8 +72,7 @@ def load(path: str) -> BaseRecommender:
     :return: Restored trained model
     """
     spark = State().session
-    with open(join(path, "init_args.json"), "r") as json_file:
-        args = json.load(json_file)
+    args = spark.read.json(join(path, "init_args.json")).first().asDict(recursive=True)
```

Comment:
what is `first` for?

Reply:
> what is `first` for?

this is needed to take first row from spark json file. "init args" are represented as a sequence of args and are written as a row in the spark json file.